README.md

```diff
@@ -9,6 +9,40 @@ From time to time there is a need to merge the changes from the original repo in
 | --- | --- |
 |**Original corresponding commit hash:**| `d50aef95520df4216c638495a6049125c00742cb` |
 
+## Update Guide
+
+- consider the new Base Git Tag from the official repo onto which we'll rebase our changes
+  
+  > **NOTE** For simplicity, let's consider `2.11.2` is the tag we want to rebase onto.
+  
+- sync our fork to include that Git Tag. On local, after cloning UiPath/FreeRdp, execute the following:
+
+  ```pwsh
+  git fetch --tags --all   # this will fetch all tags from all upstreams, including from FreeRdp/FreeRdp
+  git push --tags          # this will push the new tags to UiPath/FreeRdp
+  ```
+- checkout the `uipath` branch
+- ‼️ create a new support branch with the name format
+
+  ```
+  robot/support/before_update_to_<NEW BASE GIT TAG>
+  ```  
+  (i.e. `robot/support/before_update_to_2_11_2`)
+  
+- create **and checkout** a new branch that will be used as a work branch (i.e. `feat/update_to_2_11_2`)
+
+  > **NOTE** Right at this instant, this and the support branch should point to the same Git Commit as the `uipath` branch.
+  
+- identify the Commit Hash, which is the parent of our 1st customization Commit
+
+  > **NOTE** At the time of writing, our 1st customization Commit's message's 1st line is `Add uipath changes from previous version (2.0.0-rc3)`
+
+- Make sure you've checked out the work branch (i.e. `feat/update_to_2_11_2), and run **git rebase --onto**, so that all UiPath customization commits are replayed onto the new Base Git Tag:
+
+  ```pwsh
+  git rebase --onto <NEW BASE Git Tag> <Our 1st customization Commit's Hash> <The work branch>
```

Comment:
What should be done after rebase onto?

Reply:
Maybe squash all customisation commits? This would make the rebase easier.

Reply:
I would not squash our modification history.

Your 2nd comment is legitimate. However, the purpose of this PR is to provide the 1st iteration in the writing of the guide for **Updating the Fork**.

Instead of wasting a lot of time and coming up with an arbitrary plan for the final steps, it was decided (cc: @vuplea) that there will be a complementary PR the next time we need to Update the Fork. At that time, the final steps will be determined and written down.